lib/single-new-eks-opensource-observability-pattern/graviton-index.ts

```diff
@@ -92,7 +92,9 @@ export default class SingleNewEksGravitonOpenSourceObservabilityPattern {
             .account(account)
             .region(region)
             .version('auto')
-            .enableOpenSourcePatternAddOns(ampAddOnProps,"v1.27.1-eksbuild.1")
+            .withAmpProps(ampAddOnProps)
+            .withKubeProxyProps({},"v1.27.1-eksbuild.1")
```

Comment:
pls check this @elamaran11

Reply:
passing {} is fine?

Reply:
`.withKubeProxyProps({},"v1.27.1-eksbuild.1")` this can be removed because default it uses "v1.27.1-eksbuild.1" with autp with your kubeproxy change